shared/chat/conversation/messages/wrapper/exploding-height-retainer/index.desktop.js

```diff
@@ -24,6 +23,8 @@ type State = {
   height: number,
 }
 class ExplodingHeightRetainer extends React.PureComponent<Props, State> {
+  _boxRef = React.createRef()
+  setHeight: () => void
```

Comment:
why is this here?

Reply:
I don't recognize that, no idea. Will nuke.